src/gui/guiFormSpecMenu.h

```diff
@@ -425,6 +425,9 @@ class GUIFormSpecMenu : public GUIModalMenu
 		std::unordered_map<std::string, GUITable::DynamicData> table_dyndata;
 	};
 
+	static const std::unordered_map<std::string,
+		void (GUIFormSpecMenu::*)(GUIFormSpecMenu::parserData *data, const std::string &description, const std::string &type)> element_parsers;
```

Comment:
not sure how well it fits but a `std::function` would be more C++-like

could even couple that with `std::variant` to not have the useless third parameter